apps/web/src/screens/Setting/components/index.ts

```diff
@@ -0,0 +1,7 @@
+export { Delimiter } from './Delimiter';
```

Comment:
if those component could be reuse in other place like Mobile, then it should be in `packages/app`.